aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Choose an ASP.NET Core UI
+author: wadepickett
+description: Understand when to use the various ASP.NET Core web UI technologies Microsoft provides and supports.
+ms.author: wpickett
+ms.date: 06/30/2021
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: tutorials/choose-web-ui
+---
+# Choose an ASP.NET Core web UI
+
+ASP.NET Core is a complete end to end UI framework.  You choose which functionalities to combine that fit your web UI app needs.
+
+Start by determining if your web UI app will primarily benefit from pre-rendering UI on the server, or rendering UI at the client browser. Then review the server rendered or client rendered ASP.NET Core project types to find which best fit your apps needs. 
+
+## Choose an ASP.NET Core web UI technology
+
+Use the following flowchart to select an ASP.NET Core web UI technology:
+
+![Decision tree for ASP.NET Core web UI technology](choose-web-ui/_static/choose_ui.png)
+
+## Choose server rendered UI or client rendered UI
+
+There are two general approaches to building modern web UI:
```

Comment:
```suggestion
There are two general approaches to building modern web UI with ASP.NET Core:
```

There are other ways to build web UI outside of ASP.NET Core, like using static site generators.